assignments/lambda-classes.js

```diff
@@ -1 +1,146 @@
 // CODE here for your Lambda Classes
+class Person {
+    constructor(personAttributes){
+        this.name = personAttributes.name;
+        this.age = personAttributes.age;
+        this.location = personAttributes.location;
+        this.gender = personAttributes.gender;
+    }
+    speak() {
+        console.log(`Hello my name is ${this.name}, I am from ${this.location}`);
+    }
+}
+
+class Instructor extends Person {
+    constructor(instructorAttributes){
+        super(instructorAttributes);
+        this.specialty = instructorAttributes.specialty;
+        this.favLanguage = instructorAttributes.favLanguage;
+        this.catchPhrase = instructorAttributes.catchPhrase;
+    }
+    demo(subject){
+        console.log(`Today we are learning about ${subject}`);
+    }
+    grade(student, subject){
+        console.log(`${student.name} receives a perfect score on ${subject}`);
+    }
+    updateGrade(student){
+        let gradeChange = Math.floor(Math.random() * (100 - (-100)) + -100);
+        let newGrade = student.grade + gradeChange
```

Comment:
If you have time, and if you are up for a challenge, see if you could set up an object literal in the constructor of the class that would in theory function as a temporary class-based storage for a grade, or even an array of grades.  Just a fun challenge if you have time after the Sprint and want an interesting problem to solve for practice.